Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml.cs

```diff
@@ -1054,9 +1529,96 @@ private void PerformScreenSwipe(eSwipeSide swipeSide, double impact = 1)
         }
         #endregion Functions
 
-        private void xRecordingImage_Click(object sender, RoutedEventArgs e)
+        GenericWindow genWin;
+
+        private void OpenPopUpWindow(string content, string title)
         {
+            string tempFilePath = GingerCoreNET.GeneralLib.General.CreateTempTextFile(content);
+            //string filePath = Path.Combine(Directory.GetCurrentDirectory(), "temp.json");
+            //File.WriteAllText(filePath, content);
+            if (System.IO.File.Exists(tempFilePath))
+            {
+                Application.Current.Dispatcher.Invoke(() =>
+                {
+                    DocumentEditorPage docPage = new DocumentEditorPage(tempFilePath, enableEdit: false, UCTextEditorTitle: string.Empty);
+                    docPage.Width = 800;
+                    docPage.Height = 800;
+                    GingerCore.General.LoadGenericWindow(ref genWin, App.MainWindow, eWindowShowStyle.Free, title, docPage);
+
+                });
+
+                System.IO.File.Delete(tempFilePath);
+                return;
+            }
+        }
+
+
 
+        private void xAgentBtn_Click(object sender, RoutedEventArgs e)
+        {
+            Application.Current.Dispatcher.Invoke(() =>
+            {
+                AgentEditPage agentEditPage = new AgentEditPage(mAgent);
+                agentEditPage.Width = 800;
+                agentEditPage.Height = 800;
+                ((Grid)((ScrollViewer)agentEditPage.Content).Content).IsEnabled = false;
+                GingerCore.General.LoadGenericWindow(ref genWin, App.MainWindow, eWindowShowStyle.Free, mAgent.Name, agentEditPage);
+            });
+        }
+
+        private void xRamUsageBtn_Click(object sender, RoutedEventArgs e)
+        {
+            OpenPopUpWindow(DictionaryToString(mDeviceMemoryInfo).Replace(", ", Environment.NewLine), "Full device's Memory Information");
```

Comment:
need to change title, this is not "Full device.." but only App Memory and CPU- need to change for both